src/lib/Toast/properties.ts

```diff
@@ -0,0 +1,23 @@
+export type ToastType = 'success' | 'error' | 'info' | 'warn';
+
+export type ToastProperties = {
+  type: ToastType;
+  message: string;
+  backgroundColor?: string;
+  textColor?: string;
+  leftIcon?: string | null;
+  borderRadius?: string;
+  duration?: number;
+  showRightIcon?: boolean;
+};
+
+export const defaultToastProperties: ToastProperties = {
+  type: 'success',
+  message: 'Toast',
+  backgroundColor: '#24AA5A',
```

Comment:
All styling parameters must be passed in css variables.

Lets add primitive classes available for `success`, `failure` & `warning` in css.
The same class can be applied based upon type of the toast.

Reply:
updated.